nvflare/tool/job/job_cli.py

```diff
@@ -183,12 +176,25 @@ def get_src_template(cmd_args) -> Optional[str]:
     return None
 
 
-def remove_extra_file(config_dir):
-    extra_file = [JOB_INFO_MD, JOB_INFO_CONF]
+def remove_non_python_files(custom_dir):
+    for root, dirs, files in os.walk(custom_dir):
+        for f in files:
+            file_path = os.path.join(root, f)
+            if not f.endswith(".py") and os.path.exists(file_path):
+                os.remove(file_path)
+        for d in dirs:
+            if d == "__pycache__" or d.endswith(".pyc"):
+                shutil.rmtree(d)
+
+
+def remove_extra_files(config_dir):
+    extra_file = [JOB_INFO_MD, JOB_INFO_CONF, "__init__.py", "__pycache__"]
```

Comment:
Not 100% sure if `__init__.py` is extra file though.
Sometimes people do some import there?